src/validations/mergeValidations.js

```diff
@@ -4,9 +4,12 @@ import flatten from 'lodash-es/flatten';
 import flatMap from 'lodash-es/flatMap';
 import sortBy from 'lodash-es/sortBy';
 import omit from 'lodash-es/omit';
+import uniqWith from 'lodash-es/uniqWith';
+
 
 function filterErrors(errors) {
-  const groupedErrors = groupBy(errors, 'reason');
+  const dedupedErrors = uniqWith(errors, e => e.reason && e.row);
```

Comment:
I don’t think this is doing quite what you intend—[`uniqWith`](https://lodash.com/docs/4.17.11#uniqWith) passes two arguments to the callback, which are two values to be compared. So, your function should take two parameters, and then return `true` if the `reason` and `row` properties have the same value across those two parameters.

Reply:
hi thanks for this I am testing now